Packs/PAN-OS/ReleaseNotes/1_8_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Fixed an issue where the **panorama-get-url-category** would fail in case the length of the url is longer than 1278 characters.
```

Comment:
```suggestion
- Fixed an issue where the ***panorama-get-url-category*** would fail in case the length of the URL is longer than 1278 characters.
```